Unfortunately, this seems to fail on Linux:
```

[ERROR]   CompressedApplicationInputStreamTest.require_that_gnu_tared_file_can_be_unpacked:95->assertTestApp:67 expected:<3> but was:<0>
[ERROR]   CompressedApplicationInputStreamTest.require_that_nested_app_can_be_unpacked:107 expected:<5> but was:<0>

```

--- a/configserver/src/test/java/com/yahoo/vespa/config/server/application/CompressedApplicationInputStreamTest.java
+++ b/configserver/src/test/java/com/yahoo/vespa/config/server/application/CompressedApplicationInputStreamTest.java
@@ -142,7 +142,7 @@ public class CompressedApplicationInputStreamTest {
 
     private File createTarGz(String appDir) throws IOException, InterruptedException {
         File tmpTar = Files.createTempFile(temporaryFolder.getRoot().toPath(), "myapp", ".tar").toFile();
-        Process p = new ProcessBuilder("tar", "-C", appDir, "-cvf", tmpTar.getAbsolutePath(), ".").start();
+        Process p = new ProcessBuilder("tar", "--disable-copyfile", "-C", appDir, "-cvf", tmpTar.getAbsolutePath(), ".").start();
         p.waitFor();
         p = new ProcessBuilder("gzip", tmpTar.getAbsolutePath()).start();
         p.waitFor();

--- a/configserver/src/test/java/com/yahoo/vespa/config/server/application/CompressedApplicationInputStreamTest.java
+++ b/configserver/src/test/java/com/yahoo/vespa/config/server/application/CompressedApplicationInputStreamTest.java
@@ -141,8 +141,10 @@ public class CompressedApplicationInputStreamTest {
     }
 
     private File createTarGz(String appDir) throws IOException, InterruptedException {
+        var noMacMetadataOption = System.getProperty("os.name").startsWith("Mac OS X") ? "--no-mac-metadata" : "";
+
         File tmpTar = Files.createTempFile(temporaryFolder.getRoot().toPath(), "myapp", ".tar").toFile();
-        Process p = new ProcessBuilder("tar", "--disable-copyfile", "-C", appDir, "-cvf", tmpTar.getAbsolutePath(), ".").start();
+        Process p = new ProcessBuilder("tar", noMacMetadataOption, "-C", appDir, "-cvf", tmpTar.getAbsolutePath(), ".").start();
         p.waitFor();
         p = new ProcessBuilder("gzip", tmpTar.getAbsolutePath()).start();
         p.waitFor();

